.circleci/config.yml

```diff
@@ -188,23 +188,23 @@ jobs:
             echo "Your dependencies are secure enough: $SUMMARY"
             exit 0
 
-  #  testE2e:
-  #    executor: web
-  #    environment:
-  #      - BINARY_PATH: "/tmp/gaia/builds/Gaia/linux_amd64/gaiacli"
-  #      - NODE_BINARY_PATH: "/tmp/gaia/builds/Gaia/linux_amd64/gaiad"
-  #    steps:
-  #      - checkout
-  #      - attach_workspace:
-  #          at: *GAIA
-  #      - yarn-install
-  #      - run:
-  #          name: Test
-  #          command: yarn run test:e2e
-  #          no_output_timeout: 120
-  #      - store_artifacts:
-  #          path: testArtifacts
-  #          when: on_fail
+  testE2e:
+    executor: web
+    environment:
+      - BINARY_PATH: "/tmp/voyager/linux_amd64/gaiacli"
```

Comment:
`/tmp/voyager` is the workspace, this is why the gaia files are attached there. we need to clean the paths up.